R/estimate_KM.R

```diff
@@ -5,6 +5,7 @@
 #'    argument are removed.
 #'    Alternatively, PARAM/PARAMCD can be used in the \code{strata} argument. \cr
 #'    The result is an object of class \code{survfit} which can be used in downstream functions and methods that rely on the \code{survfit} class.
+#'    When stratum are present, the returned survfit object is supplemented with the a named list of the stratum and associated label, if present.
```

Comment:
Documentation is clear now: stratum -> strata

Reply:
updated.